PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+Please employ to following template as the base structure of Pull Requests for **Alloy-Editor** project. Pull Requests not incorporating the sections below will be reqarded as unsatisfying and rejected. 
+
+## Prelude
+Preliminary history of the issue related this very Pull Request. Share any information that could ease the understanding of why this issue exists and what measures have been administered to address it.
```

Comment:
Maybe missing **to** connector? -> `history of the issue related **to** this very Pull Request`